### PR TITLE
Add general layout for design page

### PR DESCRIPTION
**Pull request by yusufsheiqh**
## What

- Add design page in its respective directory.
- Add title HTML element to page
- Add placeholders for skills sections

## Why

This allows us to have a design page as it is a component/discipline of delivering digital services.

### CODE REVIEW

File: design/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+</div>
+
+<div class="row">
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">User research</h3>
+      <p class="card-text">
+        Learn various techniques on how to conduct iterative user research effectively.
+      </p>
+      <a href="/" class="btn btn-success">Learn more</a>
```
Comment (faissalMT) [inline design/index.html]: You can deaden links with `href="#"`

File: design/index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    Coming soon!
+  </p>
+</div>
+
+<div class="row">
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">Placeholder 1</h3>
+      <p class="card-text">
+        <i>Coming soon!</i>
```
Comment (faissalMT) [inline design/index.html]: You should only be using `<i>` for things like technical terms or specialist language, `<em>` should instead be used for emphasis.

File: design/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      <a href="/" class="btn btn-success">Learn more</a>
+    </div>
+  </div>
+</div>
+<div class="row">
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">Personas</h3>
+      <p class="card-text">
+        Create fictional characters to identify what your users needs so your designs are user centred.
```
Comment (faissalMT) [inline design/index.html]:
```suggestion
        Create fictional characters based on user needs to center your designs around.
```

File: design/index.html (hunk 1 above)
Comment (yusufsheiqh) [reply design/index.html]: Resolved in https://github.com/madetech/learn/pull/160/commits/e4f752906ba74a8140cf205bc07e24f3150633e5.

File: design/index.html (hunk 2 above)
Comment (yusufsheiqh) [reply design/index.html]: Resolved in https://github.com/madetech/learn/pull/160/commits/acdcca615cc64f35b2fbf44c9bf55930ba8e38c7.

**Pull request merged**